Makefile.in

```diff
@@ -1,17 +1,26 @@
-STRIP=@STRIP@
-OBJCOPY=@OBJCOPY@
-OBJDUMP=@OBJDUMP@
+STRIP ?= @STRIP@
+OBJCOPY ?= @OBJCOPY@
+OBJDUMP ?= @OBJDUMP@
```

Comment:
Changing these to `?=` could be nice but could also cause problems if one has a polluted environment.  Thoughts?